src/server/Storages/usersStore.ts

```diff
@@ -0,0 +1,24 @@
+import "sequelize";
+import { User } from '../../lib/model';
+import { sequelize } from "../database/database";
+import { MUser } from "../database/models/MUser";
+
+export function getAll(): Promise<User[]> {
+    return sequelize.sync()
+        .then(() => {
```

Comment:
after you move `sequelize.sync()` this could be written in one line, just `return UserModel.findAll()`;